developer/docs/arvo/http.md

```diff
@@ -95,47 +113,46 @@ r/cage}` from `zuse`.
 > verify that the unit is not null (for example with `?~`). See
 > `++need`.
 
-A `purl` is a parsed url structure, which can be created with `++epur`,
-which is a function that takes a url as text and parses it into a `(unit
-purl)`.  Thus, the result is null if and only if the url is malformed.
-'Purl' is also the mark which will be applied to this result.  
+A `purl` is a parsed url structure, which can be created with `++de-purl`, which 
+is a function that takes a url as text and parses it into a `(unit purl)`. 
+Thus, the result is null if and only if the url is malformed. `&purl` is also 
+the mark which will be applied to this result.  
 
-When you send this request, you can expect a `%sigh` with the
-response, which we handle later on in `++sigh-httr`.
+When you send this request, you can expect a `%sigh` with the response, which 
+we handle later on in `++sigh-httr`.
 
-For `%wait`, you just pass a [`@da`]() (absolute date), and arvo will
-produce a `%wake` when the time comes.
+For `%wait`, you just pass a [`@da`](/../../hoon/library/3c/) (absolute date), 
+and arvo will produce a `%wake` when the time comes.
```

Comment:
We should also capitalize Nock, Hoon and Arvo everywhere we see it. Not hugely important, but we should be consistent.